PhysicsTools/PatAlgos/python/patTracksToPackedCandidates_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+patTracksToPackedCandidates = cms.EDProducer('PATTracksToPackedCandidates',
+  srcTracks = cms.InputTag('hiConformalPixelTracks'),
+  srcPrimaryVertices = cms.InputTag('offlineSlimmedPrimaryVertices'),
+  dzSigCut = cms.double(10),
+  dxySigCut = cms.double(25),
+  dzSigHP = cms.double(7), #to set HighPurity flag for pixel tracks
```

Comment:
if these comments are useful, add them in .cc as e.g. ```desc.add<double>("dzSigHP", 7.0)->setComment("to set HighPurity flag for pixel tracks");```

Reply:
This comment is included in this commit: https://github.com/cms-sw/cmssw/pull/30313/commits/73d82c08c6f959cb9762d7e040eefb75bffe6950